doc/dev/release.md

```diff
@@ -306,11 +307,24 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**Note:**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
 You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
 
-### (Post-release) Updating the operator-sdk-samples repo
+### 6. Update the Upgrade version guide document
```

Comment:
This step needs to be done _before_ we create a release tag, then the `See how to upgrade your project...` release note needs to be added when we're creating the release notes initially (since this upgrade guide section is tagged in the release).